ci-operator/step-registry/hypershift/kubevirt/create/hypershift-kubevirt-create-chain.yaml

```diff
@@ -41,65 +31,6 @@ chain:
       echo "Cluster became available, creating kubeconfig"
       bin/hypershift create kubeconfig --name=${KUBEVIRT_CLUSTER_NAME} >${SHARED_DIR}/nested_kubeconfig
 
-      # Enable wildcard routes on the management cluster
-      oc patch ingresscontroller -n openshift-ingress-operator default --type=json -p \
-        '[{ "op": "add", "path": "/spec/routeAdmission", "value": {wildcardPolicy: "WildcardsAllowed"}}]'
-
-      # Waiting for the default NodePort service to be created on the hosted cluster
-      until \
-        oc --kubeconfig ${SHARED_DIR}/nested_kubeconfig get service router-nodeport-default -n openshift-ingress
-      do
-        echo "router-nodeport-default service wasn't created yet on the hosted cluster. Retrying in 30 seconds."
-        sleep 30s
-      done
-
-      HTTPS_NODEPORT=$(oc --kubeconfig ${SHARED_DIR}/nested_kubeconfig get services -n openshift-ingress router-nodeport-default -o wide | awk '{print $5}' | awk -F "443:" '{print $2}' | awk -F "/" '{print $1}' | tr -d '[:space:]')
-
-      # Create a service for ingress to the nested cluster on the node's VMIs
-      oc create -f - <<EOF
-      apiVersion: v1
-      kind: Service
-      metadata:
-        labels:
-          app: ${KUBEVIRT_CLUSTER_NAME}
-        name: apps-ingress
-        namespace: ${CLUSTER_NAMESPACE}
-      spec:
-        internalTrafficPolicy: Cluster
-        ipFamilies:
-          - IPv4
-        ipFamilyPolicy: SingleStack
-        ports:
-          - name: https-443
-            port: 443
-            protocol: TCP
-            targetPort: ${HTTPS_NODEPORT}
-        selector:
-          kubevirt.io: virt-launcher
-        sessionAffinity: None
-        type: ClusterIP
-      EOF
-
-      # Create a Route for the ingress subdomain of the nested cluster
-      oc create -f - <<EOF
-      apiVersion: route.openshift.io/v1
-      kind: Route
-      metadata:
-        name: ${KUBEVIRT_CLUSTER_NAME}-443
-        namespace: ${CLUSTER_NAMESPACE}
-      spec:
-        host: data.apps.${CLUSTER_BASE_DOMAIN}
-        wildcardPolicy: Subdomain
-        tls:
-          termination: passthrough
-        port:
-          targetPort: https-443
-        to:
-          kind: Service
-          name: apps-ingress
-          weight: 100
-      EOF
-
       echo "Waiting for all VMIs in ${CLUSTER_NAMESPACE} to reach vmi.status.phase=Running"
       oc wait -n ${CLUSTER_NAMESPACE} VirtualMachineInstance --all --for=jsonpath='{.status.phase}'=Running --timeout 30m
```

Comment:
```
+ echo 'Waiting for all VMIs in clusters-kv-nested to reach vmi.status.phase=Running'
Waiting for all VMIs in clusters-kv-nested to reach vmi.status.phase=Running
+ oc wait -n clusters-kv-nested VirtualMachineInstance --all '--for=jsonpath={.status.phase}=Running' --timeout 30m
error: no matching resources found
```
We should add some checks before oc wait

Reply:
ah, yes i see that breaks. We probably don't even need to wait for the VMIs to come online. Waiting on the nodepool should be sufficient. I updated the code to simply print out the vmi status for debug while we wait for nodepool to finish.